CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing and development
+
+## Prerequisites
+
+* Minikube (or any other K8s cluster running)
+* kubectl
+* helm
+* Skaffold
+* nodejs
+
+## Running
+
+Start Minikube (or any other cluster of your choice):
+
+```bash
+minikube start --uuid 00000000-0000-0000-0000-000000000001 --extra-config=apiserver.enable-admission-plugins=ValidatingAdmissionWebhook
```

Comment:
Why would one need to specify the UUID?

Reply:
AFAIU it keeps the same IP address (`Provide VM UUID to restore MAC address (hyperkit driver only)` as in docs) when recreating the cluster. But... doesn't seem useful for this setup since is hyperkit only and I'm on Linux which does not support it (so I never needed it) 😅 Will get rid of it 👍

Reply:
Fixed in https://github.com/kubeshop/monokle-admission-controller/pull/8/commits/0dc0c23cfc657100c8e9879d893507ee5cbfade0